released/packages/coq-ext-lib/coq-ext-lib.0.11.3/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+maintainer: "gmalecha@gmail.com"
+homepage: "https://github.com/coq-community/coq-ext-lib"
+dev-repo: "git+https://github.com/coq-community/coq-ext-lib.git"
+bug-reports: "https://github.com/coq-community/coq-ext-lib/issues"
+authors: ["Gregory Malecha"]
+license: "BSD"
```

Comment:
```suggestion
license: "BSD-2-Clause-FreeBSD"
```
@liyishuai can you confirm that this is the intended license? https://spdx.org/licenses/BSD-2-Clause-FreeBSD.html

Reply:
Yes, it is. I'll update the coq-ext-lib repo to use this standardized identifier accordingly.